README.md

```diff
@@ -95,7 +97,7 @@ livereload          Livereload server port. Default: 35729
 host                Host name. If need to be served
 port                Port number. If need to be served
 staticAssetsPath    The root path of the static files served by the webserver, by default it is the plugin folder
-specRunner          The path of the SpecRunner.html file, by default it is the plugin folder / SpecRunner.html
+specRunner          The path to a SpecRunner.html file (relative to staticAssetsPath), if you wish to use your own runner file when serving instead of using the generated runner. This file will be used as-is, not modified by gulp-jasmine-livereload-task.
```

Comment:
This is primarily clarification — the behavior is very similar to what it was before. See the main PR for comments on the difference.